app/assets/images/payment_icons/ideal.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg width="38" height="24" viewBox="0 0 38 24" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="pi-bancontact"><title id="pi-bancontact">Bancontact</title><path fill="#F5F5F5" d="M0 0h38v24H0z"/><path d="M0 1.78v20.44C0 23.2 1.117 24 2.483 24h17.045C32.414 24 38 18.832 38 11.973 38 5.15 32.414 0 19.528 0H2.483C1.117 0 0 .8 0 1.78z" fill="#fff"/><path d="M11.409 5.017v15.105h9.174c8.33 0 11.942-3.372 11.942-8.14 0-4.563-3.612-8.104-11.942-8.104h-7.585c-.882 0-1.59.516-1.59 1.14z" fill="#C06"/><path d="M19.528 22.372H4.705c-1.316 0-2.384-.765-2.384-1.708V3.344c0-.942 1.068-1.707 2.384-1.707h14.823c14.065 0 16.163 6.485 16.163 10.345 0 6.698-5.748 10.39-16.163 10.39zM4.705 2.206c-.881 0-1.589.507-1.589 1.139v17.32c0 .63.708 1.138 1.589 1.138h14.823c9.906 0 15.368-3.487 15.368-9.82 0-8.505-9.633-9.777-15.368-9.777H4.705z" fill="#000"/><path d="M14.636 9.945c.323 0 .621.036.907.107.285.071.521.187.732.329.199.151.36.338.484.551.112.223.174.48.174.783 0 .267-.05.507-.136.73-.1.222-.236.418-.423.578-.186.16-.422.285-.707.383-.286.088-.62.142-1.006.142H12.49V9.936h2.147v.01zm-.074 2.944c.161 0 .31-.017.472-.053.149-.035.285-.098.397-.187a1.05 1.05 0 00.285-.338c.075-.142.112-.302.112-.507a1.51 1.51 0 00-.074-.489.894.894 0 00-.249-.374 1.161 1.161 0 00-.447-.24 2.487 2.487 0 00-.682-.08h-.795v2.277h.98v-.008zm6.853-2.944v.667h-2.657v.774h2.445V12h-2.445v.88h2.719v.668h-3.824V9.936h3.762v.01zm3.798 0l1.887 3.612h-1.154l-.385-.8h-1.887l-.397.8h-1.118l1.9-3.612h1.154zm.062 2.215l-.633-1.325h-.012l-.658 1.325h1.303zM28.9 9.945v2.944h2.458v.668h-3.563V9.945H28.9z" fill="#fff"/><path d="M7.262 13.414c1.282 0 2.322-.744 2.322-1.663s-1.04-1.663-2.322-1.663-2.321.744-2.321 1.663 1.04 1.663 2.321 1.663zM9.013 20.122c-1.95 0-3.513-1.13-3.513-2.518v-1.966c0-.694.782-1.263 1.762-1.263.969 0 1.763.56 1.763 1.263v4.484h-.012z" fill="#000"/></svg>
```

Comment:
See my previous comment. We should also apply the same changes here.

Reply:
uopdated.